ext/opcache/ZendAccelerator.c

```diff
@@ -624,6 +632,7 @@ static void accel_copy_permanent_strings(zend_new_interned_string_func_t new_int
 
 		if (ce->name) {
 			ce->name = new_interned_string(ce->name);
+			ZEND_ASSERT(ZSTR_HAS_CE_CACHE(ce->name));
```

Comment:
This assertion is failing and causing all phpdbg tests to fail in tracing jit builds